src/EnergyPlus/HVACVariableRefrigerantFlow.cc

```diff
@@ -5812,9 +6179,33 @@ namespace HVACVariableRefrigerantFlow {
         // TerminalUnitList(TUListIndex)%HRCoolRequest(IndexToTUInTUList) - TU will operate in this mode if heat recovery is used
         // TerminalUnitList(TUListIndex)%HRHeatRequest(IndexToTUInTUList) - TU will operate in this mode if heat recovery is used
 
-        QZnReq = ZoneSysEnergyDemand(VRFTU(VRFTUNum).ZoneNum).RemainingOutputRequired;
+        if (VRFTU(VRFTUNum).zoneSequenceCoolingNum > 0 && VRFTU(VRFTUNum).zoneSequenceHeatingNum > 0 and VRFTU(VRFTUNum).isInAirLoop) {
```

Comment:
Air loop equipment use SequencedOutputRequired, zone equipment use RemainingOutputRequired, set point based equipment (OA sys equip is always SP controlled) use a SP based load calculated previously.

Reply:
Need to change and to && for consistency